controller/model/testing.go

```diff
@@ -120,20 +120,18 @@ func (ctx *TestContext) GetFingerprintGenerator() cert.FingerprintGenerator {
 }
 
 func NewTestContext(t *testing.T) *TestContext {
-	return NewTestContextWithDb(t, "")
-}
-
-func NewTestContextWithDb(t *testing.T, dbPath string) *TestContext {
 	context := &TestContext{
 		TestContext:     persistence.NewTestContext(t),
 		metricsRegistry: metrics.NewRegistry("test", nil),
 		hostController: &testHostController{
 			closeNotify: make(chan struct{}),
 		},
 	}
-	context.InitWithDbFile(dbPath)
```

Comment:
This line was causing init to be called twice in the session enforcer tests (the only user of this function atm) and was causing a double initialization of the underlying stores. The direct result was two EventualEventers running at once which was very confusing to debug. 

Either way, this change makes it so that the TestContext is set up like all the other ones: 1) create via New 2) Init() 3) defer Cleanup().